.github/workflows/ci.yml

```diff
@@ -36,6 +36,9 @@ jobs:
 
       - run: yarn check-dependencies
 
+      - name: 'Check version plans'
+        run: yarn nx release plan:check
```

Comment:
nx is used for releases, so we need it to be installed -> removing artificial ci-change workflow